tests/backup/backup_basic_test.go

```diff
@@ -156,6 +157,15 @@ var _ = BeforeSuite(func() {
 			globalAWSBucketName = fmt.Sprintf("%s-%s", globalAWSBucketPrefix, bucketNameSuffix)
 			CreateBucket(provider, globalAWSBucketName)
 			log.Infof("Bucket created with name - %s", globalAWSBucketName)
+			_, _, _, _, _, _, ssePolicySid, sseEncryptionPolicy := s3utils.GetAWSDetailsFromEnv()
```

Comment:
I think it better to create a struct called `S3SSEInfo` and add the three parameters `sseEncryptionPolicy`, `sseType` and `ssePolicySid` to it. A function returning 8 things is not good practice.

Reply:
done created seperate function